### PR TITLE
ANC Logic - prescribe Mebendazole

### PR DESCRIPTION
**Pull request by anvmn**
#113 
#160

### CODE REVIEW

**Comment by anvmn** [comment]
### Before 24 weeks EGA: 
Selection_492

### After 24 weeks EGA: 
Selection_491

**Comment by anvmn** [comment]
Peek 2022-02-11 17-04

**Comment by anvmn** [comment]
Peek 2022-02-11 21-00

**Comment by anvmn** [comment]
Peek 2022-02-13 14-58

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Backend/Measurement/Model.elm [hunk 1]
```diff
@@ -583,7 +583,12 @@ type alias MedicalHistory =
 
 type MedicationSign
     = IronAndFolicAcidSupplement
+      -- This option is depricated. However, we keep it, to support
```
Comment (nedSaf) [inline client/src/elm/Backend/Measurement/Model.elm]:
```suggestion
      -- This option is deprecated. However, we keep it, to support
```

File: client/src/elm/Pages/PrenatalActivity/Utils.elm [hunk 2]
```diff
@@ -284,6 +266,77 @@ expectNextStepsTasks currentDate data task =
...
+                                    |> Maybe.withDefault False
+                            )
+                            assembled.nursePreviousMeasurementsWithDates
+                            |> List.isEmpty
+                in
+                -- Starting EGA week 24.
+                (egaInWeeks >= 24)
+                    && -- Previous variation had a question about deworming pill,
+                       -- which is actually Menendazole, or something similar.
+                       -- If somwhere during previous encounters patient stated that
```
Comment (nedSaf) [inline client/src/elm/Pages/PrenatalActivity/Utils.elm]:
```suggestion
                       -- If somewhere during previous encounters patient stated that
```

File: client/src/elm/Pages/PrenatalActivity/Utils.elm [hunk 3]
```diff
@@ -284,6 +266,77 @@ expectNextStepsTasks currentDate data task =
...
+                       -- deworming pill was given, we do not ask about Mebendazole
+                       dewormingPillNotGiven
+                    && -- Mebendazole was not prescribed during current pregnancy.
```
Comment (nedSaf) [inline client/src/elm/Pages/PrenatalActivity/Utils.elm]:
```suggestion
                    && -- Mebendazole was not prescribed during the current pregnancy.
```

File: client/src/elm/Pages/PrenatalActivity/View.elm [hunk 4]
```diff
@@ -2799,6 +2798,92 @@ viewNewbornEnrolmentForm language currentDate assembled =
...
+                    [ viewQuestionLabel language Translate.WhyNot
+                    , viewCheckBoxSelectInput language
+                        [ NonAdministrationLackOfStock, NonAdministrationKnownAllergy, NonAdministrationPatientUnableToAfford ]
+                        [ NonAdministrationPatientDeclined, NonAdministrationOther ]
+                        currentValue
+                        (SetMedicationDistributionAdministrationNote currentValue medication)
+                        Translate.AdministrationNote
+                    ]
+
+                -- When answer for medication administartion is Yes, we clean the reason for not administering the medication.
```
Comment (nedSaf) [inline client/src/elm/Pages/PrenatalActivity/View.elm]:
```suggestion
                -- When the answer for medication administration is Yes, we clean the reason for not administering the medication.
```

**Review by nedSaf — APPROVED** [review]
👍  LGTM, just some minor typos, then MWG.

**Pull request merged**